src/router/index.js

```diff
@@ -74,6 +76,11 @@ const router = new Router({
 			props: route => ({
 				rootTitle: t('photos', 'All your media'),
 			}),
+			meta: {
```

Comment:
Unfortunately that doesn't works this way. ``` props ``` are transmitted from here to component and won't be saved in the route. Because of this i need another solution which will be saved in a route.